_plugins/link_card_tag.rb

```
@@ -0,0 +1,177 @@
+      if location && !location.empty?
+        archive_url = "https://web.archive.org#{location}"
+        log_info("SavePageNow archived #{url} -> #{archive_url}")
+        archive_url
+      else
+        log_debug("archive submission returned no location for #{url}")
+        nil
+      end
+    rescue StandardError => e
+      log_debug("archive submission error for #{url}: #{e.message}")
+      nil
+    end
```

RuboCop (1.81.7)

[convention] Assignment Branch Condition size for `submit_archive` is too high. [<8, 21, 6> 23.26/23]

(Metrics/AbcSize)